pipelines/skylab/optimus/Optimus.wdl

```diff
@@ -237,6 +239,12 @@ workflow Optimus {
     File gene_metrics = GeneMetrics.gene_metrics
     File? cell_calls = RunEmptyDrops.empty_drops_result
     File? aligner_metrics = MergeStarOutputs.cell_reads_out
+    Array[File] soloMultiMappers = STARsoloFastq.matrix_sn_rna
```

Comment:
Is this output supposed to optional?

Reply:
I dont think so, because we do a `touch matrix_sn_rna.mtx` in the starsolofastq command, so we will always capture this as an output, even if it is empty